guppylang/prelude/quantum_functional.py

```diff
@@ -1,110 +1,134 @@
 """Guppy standard module for quantum operations."""
 
-# mypy: disable-error-code="empty-body, misc, valid-type"
-
-import typing
+from typing import no_type_check
 
+import guppylang.prelude.quantum as quantum
 from guppylang.decorator import guppy
-from guppylang.prelude._internal.compiler.quantum import HSERIES_EXTENSION
-from guppylang.prelude._internal.util import quantum_op
+
+# mypy: disable-error-code="empty-body, misc, valid-type"
+from guppylang.module import GuppyModule
 from guppylang.prelude.angles import angle
 from guppylang.prelude.builtins import owned
-from guppylang.prelude.quantum import quantum, qubit
+from guppylang.prelude.quantum import qubit
+
+quantum_functional = GuppyModule("quantum_functional")
+quantum_functional.load(angle, qubit, quantum)
 
 
-@guppy.hugr_op(quantum, quantum_op("H"))
-def h(q: qubit @ owned) -> qubit: ...
+@guppy(quantum_functional)
+@no_type_check
+def h(q: qubit @ owned) -> qubit:
```

Comment:
ugh. Does CI run formatting on this? I guess it does, so we get an error if we write `@owned`. Boo.